src/main/java/com/jxc/readapis/controllers/ArticleController.java

```diff
@@ -0,0 +1,38 @@
+package com.jxc.readapis.controllers;
+
+import com.jxc.readapis.dto.ArticleInfoDTO;
+import com.jxc.readapis.dto.UserInfosDTO;
+import com.jxc.readapis.exceptions.UserNotFoundException;
+import com.jxc.readapis.services.ArticleService;
+import com.jxc.readapis.services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/api/v1/articles", produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+public class ArticleController {
+
+    @Autowired
+    ArticleService articleService;
+
+    @GetMapping("/{email}")
+    public ResponseEntity<List<ArticleInfoDTO>> getUserArticles(@PathVariable String email) {
+        List<ArticleInfoDTO> articles = this.articleService.getUserArticles(email);
```

Comment:
Lié avec la modif sur ArticleService
`public ResponseEntity<Page<ArticleInfoDTO>> getUserArticles(@PageableDefault(size = 20) final Pageable page, @PathVariable String email) {`
`Page<ArticleInfoDTO> articles = this.articleService.getUserArticles(page, email);`